core/src/data/value/mod.rs

```diff
@@ -177,7 +177,7 @@ impl Value {
         Ok(())
     }
 
-    pub fn validate_null(&self, nullable: bool) -> Result<()> {
+    pub fn validate_null(&self, nullable: &bool) -> Result<()> {
```

Comment:
`bool` implements `Copy` trait so it does not need to be `&bool`, it would be simply enough to use `bool`